README.md

```diff
@@ -85,12 +74,11 @@ See the default action step definition:
   env:
     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
   with:
-    github-repository: "{ org }/{ repo }"
-    pr-numer: 109
+    github-repository: "{ org }/{ repo }"         # ${{ github.repository }}
```

Comment:
As I mentioned in VersionTagChecker project, this comment is little confusing. I see that the number below is the number of github.event. But for repository, I don't think it is true. It should be repository.login.owner and repository.login.name or something like it. OR it is same as repository.id.

Reply:
Fixed.